src/roles/vsd-deploy/tasks/main.yml

```diff
@@ -39,17 +39,22 @@
 - name: Set if skipping VSD deploy
   set_fact: skip_vsd_deploy="{{ vsd_versions_match is defined and vsd_versions_match and monit_result.rc == 0 }}"
 
-- name: Display if skipping VSD deploy
-  debug:
-    msg:
-      - "*************************************************"
-      - "Skipping VSD deploy because it is already running"
-      - "*************************************************"
-  when: skip_vsd_deploy
+- block:
```

Comment:
vsd-deploy should not be run at all during an inplace upgrade.  I suspect there is an issue during inventory generation at build.  This is what needs to get fixed.  Please revert these changes.